package programmingExcercises;

import exceptions.CheckArgumentException;

public class ExceptionsClient {

	public static void main(String[] args) {
		question1(args);
		
	}

	private static void question1(String... args) {
		CheckingArgument boj = new CheckingArgument();
		try {
			boj.validateNumbers(args);
		} catch (CheckArgumentException e) {
			System.out.println(e.getMessage());
		}
	}
	private static void question2(String...args) {
		
	}

}
